code/modules/reagents/reagent_containers.dm

```diff
@@ -49,11 +53,34 @@
 	if(!possible_transfer_amounts)
 		src.verbs -= /obj/item/reagent_containers/verb/set_APTFT
 	create_reagents(volume)
+	shatter_material = new
```

Comment:
This should either be new'd when we actually use it (in shatter()) or ideally we should have a materials repository like the decl repository - Because this is a singleton.

Reply:
uh oh